tests/psi4numpy/fci/input.py

```diff
@@ -0,0 +1,143 @@
+import time
+import numpy as np
```

Comment:
I'm really happy to see more of these API oriented tests appearing.  Will be really helpful as we try to push the API instead of Psithon.